src/www/services_rfc2136_edit.php

```diff
@@ -222,6 +226,14 @@
                     </td>
                   </tr>
                   <tr>
+                    <td valign="top" class="vncellreq"><?=gettext("Record Type");?> </td>
+                    <td class="vtable">
+                      <input name="recordtype" type="radio" value="A" <?php if ($pconfig['recordtype'] == "A") echo "checked=\"checked\""; ?> /> <?=gettext("A (IPv4)");?> &nbsp;
+                      <input name="recordtype" type="radio" value="AAAA" <?php if ($pconfig['recordtype'] == "AAAA") echo "checked=\"checked\""; ?> /> <?=gettext("AAAA (IPv6)");?> &nbsp;
+                      <input name="recordtype" type="radio" value="both" <?php if ($pconfig['recordtype'] == "both") echo "checked=\"checked\""; ?> /> <?=gettext("Both");?>
```

Comment:
value="" and empty($pconfig['recordtype']) is better, also the default selection should be the first in the list.